src/containers/UploadContainer.js

```diff
@@ -77,6 +77,7 @@ const UploadContainer = props => {
     >
       {(createFile, {data, error}) => (
         <FileUploadTarget
+          className={props.className}
```

Comment:
I don't think that the `UploadContainer` has a `className` prop. (We should specify `prop-types` though)